package/MDAnalysis/coordinates/PDB.py

```diff
@@ -56,44 +56,22 @@
 Implementations
 ---------------
 
-Two different implementations of PDB I/O are available: the
-":ref:`permissive<permissive>`" and the ":ref:`strict<strict>`" Reader/Writers.
-The default are the "permissive" ones but this can be changed by setting the
-flag "permissive_pdb_reader" in :data:`MDAnalysis.core.flags` (see
-:ref:`flags-label`) to ``False``::
+PDB I/O is available in the form of the Simple PDB Reader/Writers.
 
-   MDAnalysis.core.flags["permissive_pdb_reader"] = False
+..deprecated:: 0.15.0
+Readers amd writers solely available in the form of
+":ref:`permissive<permissive>`" Readers and Writers, see below.
```

Comment:
I would remove all mentions of permissive from the docs here and only mention it for the deprecations.